src/View/Components/Select.php

```diff
@@ -11,6 +11,7 @@ public function __construct(
         public bool $clearable = true,
         public bool $searchable = true,
         public bool $multiselect = false,
+        public bool $showItemsCount = true,
```

Comment:
Could you rename it to **withoutItemsCount**, so we can pass it in an easy way
```suggestion
        public bool $withoutItemsCount = false,
```
`<x-select without-items-count ... />`